internals/secrethub/inject.go

```diff
@@ -115,8 +115,12 @@ func (cmd *InjectCommand) Run() error {
 	switch cmd.templateVersion {
 	case "1":
 		parser = tpl.NewV1Parser()
+	case "v1":
```

Comment:
Did you know you can just do `case "v1", "1":`?

Reply:
No, thnx :smile:

Reply:
You're welcome